packages/mobile-signing-provider/esbuild.config.js

```diff
@@ -1,11 +1,467 @@
+                    }
+                  };
+                  throw new TypeError(s ? "Object is not iterable." : "Symbol.iterator is not defined.");
+                }
+      
+                export function __read(o, n) {
+                  var m = typeof Symbol === "function" && o[Symbol.iterator];
+                  if (!m) return o;
+                  var i = m.call(o), r, ar = [], e;
+                  try {
+                    while ((n === void 0 || n-- > 0) && !(r = i.next()).done) ar.push(r.value);
+                  }
+                  catch (error) { e = { error: error }; }
+                  finally {
+                    try {
+                      if (r && !r.done && (m = i["return"])) m.call(i);
+                    }
+                    finally { if (e) throw e.error; }
+                  }
+                  return ar;
+                }
+              `,
+              loader: 'js',
+            })
+          );
+        },
+      },
+      {
+        name: 'replace-unstorage',
+        setup(build) {
+          build.onResolve({ filter: /^unstorage$/ }, () => ({
+            path: 'virtual:unstorage',
+            namespace: 'virtual',
+          }));
+
+          build.onLoad(
+            { filter: /^virtual:unstorage$/, namespace: 'virtual' },
+            () => ({
+              contents: `
+                export function createStorage(options = {}) {
+                  return {
+                    async getItem(key) {
+                      try {
+                        return localStorage.getItem(key);
+                      } catch (e) {
+                        console.warn('Storage getItem error:', e);
+                        return null;
+                      }
+                    },
+                    async setItem(key, value) {
+                      try {
+                        localStorage.setItem(key, value);
+                      } catch (e) {
+                        console.warn('Storage setItem error:', e);
+                      }
+                    },
+                    async removeItem(key) {
+                      try {
+                        localStorage.removeItem(key);
+                      } catch (e) {
+                        console.warn('Storage removeItem error:', e);
+                      }
+                    },
+                    async clear() {
+                      try {
+                        localStorage.clear();
+                      } catch (e) {
+                        console.warn('Storage clear error:', e);
+                      }
+                    },
+                    async mount() {},
+                    async unmount() {}
+                  };
+                }
+
+                export default {
+                  createStorage
+                };
+              `,
+              loader: 'js',
+            })
+          );
+        },
+      },
+      {
+        name: 'replace-signing-key',
```

Comment:
this one, probably bad idea, but adds a lot to the bundle, shame that wallet connect libs are not using @noble/ libs